macros/schema_tests/not_constant.sql

```diff
@@ -0,0 +1,21 @@
+{% macro test_not_constant(model, arg) %}
+
+select count(*)
+from
+  (
+  select
+        count(distinct not_constant_field) count_of_distinct_rows
+    from (
+          select
+              {{ arg }} as not_constant_field
+
+          from {{ model }}
+
+          where {{ arg }} is not null
+
+         ) not_null_rows
+  ) discount_rows_count
+
+  where count_of_distinct_rows = 1
+
+{% endmacro %}
```

Comment:
I think the approach listed above will work here too! This macro is interesting in that a column full of `null` values will "pass" this test. Is that the desired behavior? I suppose it makes sense to use it in conjunction with `atleast_one` above

Reply:
Yes that was intended, I didnt want to generate false positives when we already have a separate test for it.